src/precedence.js

```diff
@@ -28,14 +35,26 @@ Precedence.prototype.calculateResult = function() {
     }
     return finalResult;
 }
+
+/**
+ * Allow calculation with precedence operations  
+ */
 Precedence.prototype.precedenceOn = function() {
     this.turnOn = true;
 }
 
+/**
+ * Add operations into operations array 
```

Comment:
Adds operation.

Operations array is internal state, which should not interest anyone who is using your methods.